### DISCUSSION TITLE
ImportError: cannot import name 'Update' from 'telegram' (E:\software\venv\Lib\site-packages\telegram\__init__.py) (venv) PS E:\software>

### DISCUSSION THREAD

**Discussion by MrDictator01** [python-telegram-bot/python-telegram-bot#3931] [Q&A]
it says  cannot import name 'Update' from 'telegram what to do

**Comment by lemontree210** [comment]
Hi! Please see [this FAQ entry](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Frequently-Asked-Questions#why-am-i-getting-importerror-cannot-import-name-xy-from-telegram).
